benchmark/util/promisify.js

```diff
@@ -0,0 +1,77 @@
+'use strict';
+
+const common = require('../common');
+const util = require('util');
+const fs = require('fs');
+const zlib = require('zlib');
+const dns = require('dns');
+
+const bench = common.createBenchmark(main, {
+  n: [4e6],
+  fn: [ 'fsStat', 'zlibGzip', 'dnsLookup', 'timersSetTimeout' ]
```

Comment:
This option would have to be reflected in `test/parallel/test-benchmark-util.js`.